AppDB/appscale/datastore/cassandra_env/cassandra_interface.py

```diff
@@ -695,28 +730,30 @@ def get_metadata(self, key):
     Returns:
       A string containing the value or None if the key is not present.
     """
-    statement = """
-      SELECT {value} FROM "{table}"
-      WHERE {key} = %s
-      AND {column} = %s
-    """.format(
+    statement = (
+      'SELECT {value} FROM "{table}" '
+      'WHERE {key} = %s '
+      'AND {column} = %s'
+    ).format(
       value=ThriftColumn.VALUE,
       table=dbconstants.DATASTORE_METADATA_TABLE,
       key=ThriftColumn.KEY,
       column=ThriftColumn.COLUMN_NAME
     )
     try:
-      results = self.session.execute(statement, (bytearray(key), key))
+      results = yield self.tornado_cassandra.execute(
+        statement, (bytearray(key), key))
     except dbconstants.TRANSIENT_CASSANDRA_ERRORS:
       message = 'Unable to fetch {} from datastore metadata'.format(key)
       logging.exception(message)
       raise AppScaleDBConnectionError(message)
 
     try:
-      return results[0].value
+      raise gen.Return(results[0].value)
     except IndexError:
-      return None
+      raise gen.Return()
```

Comment:
This can just be `return`. Though there's no difference here, `return` avoids situations where a generic `except Exception` catches gen.Return unintentionally. All of the places in this PR that use `gen.Return()` without an argument can be replaced.

Reply:
Thanks! I thought that `return` statement can't be used in generators in any form, but now I realized that `return` with no arguments is handled separately and is valid in this case.